app/src/testSpIP/java/org/dhis2/usescases/sms/domain/usecase/SendSmsUseCaseTest.kt

```diff
@@ -0,0 +1,67 @@
+package org.dhis2.usescases.sms.domain.usecase
+
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
+import org.dhis2.usescases.sms.data.model.MessageTemplate
+import org.dhis2.usescases.sms.data.model.OutboundResponse
+import org.dhis2.usescases.sms.domain.model.patient.Patient
+import org.dhis2.usescases.sms.domain.model.sms.Message
+import org.dhis2.usescases.sms.domain.model.sms.SmsResult
+import org.dhis2.usescases.sms.domain.repository.message.MessageTemplateRepository
+import org.dhis2.usescases.sms.domain.repository.patient.PatientRepository
+import org.dhis2.usescases.sms.domain.repository.preferred.PreferredLanguageRepository
+import org.dhis2.usescases.sms.domain.repository.sms.SmsRepository
+import org.junit.Before
+import org.junit.Test
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.whenever
+
+@ExperimentalCoroutinesApi
```

Comment:
It should not have changes in test because the logic is the same, please maintain the tests equal to version in 2.9.1